src/app/course/_test/token-use-snippet/token-use-snippet.component.spec.ts

```diff
@@ -19,6 +26,7 @@ describe('TokenUseSnippetComponent', () => {
             imports: [TestModule, CourseModule],
             declarations: [TokenUseSnippetComponent],
             providers: [
+                {provide: AuthenticationService, useClass: AuthenticationServiceMock},
```

Comment:
Let's have `AuthenticationServiceMock` and `AuthenticationServiceMockInstructor`